skrf/io/touchstone.py

```diff
@@ -459,68 +461,82 @@ def is_from_hfss(self):
 
         return False
 
-    def get_gamma_z0(self):
+    def get_gamma_z0_from_fid(self, fid):
         '''
-        Extracts Z0 and Gamma comments from touchstone file (is provided)
+        Extracts Z0 and Gamma comments from fid
```

Comment:
should we attempt some backward compatability here?

Reply:
I didn't change `get_gamma_z0_from_fid`, this method is new. The public method `get_gamma_z0` has not changed the interface, it returns exactly the same as before.

Reply:
ok great . thanks.